pctiler/pctiler/reader.py

```diff
@@ -33,7 +32,7 @@ def get_cache_key(
 
 
 @attr.s
-class ItemSTACReader(STACReader):
+class ItemSTACReader(PgSTACReader):
```

Comment:
Looks like the Item property on `ItemSTACReader` is now `input`, so the `_get_asset_url` implementation in this class needs to be updated e.g., `self.input.collection_id` to check the render configs.

Reply:
Good Catch 🙏